src/sql.rs

```diff
@@ -93,7 +93,7 @@ impl Sql {
                 }
             }
         }
-        res.map_err(|e| format_err!("Could not open db: {}", e))
+        res.map_err(|e| format_err!("Could not open db file {}: {}", dbfile.as_ref().to_string_lossy(), e))
```

Comment:
Shouldn't `e` be printed with `#` formatter too?

Reply:
There is no .context() here, but can't harm.